### ISSUE TITLE
Error on disabling the package

### ISSUE THREAD

**Issue by puranjayjain** [thtliife/grammar-tab-colors#2]
I'm using the following:
Atom: v1.7.4
OS: Window 10 x64 Enterprise

I'm seeing the following error each time I enable or disable the package:
```
Error deactivating package 'grammar-tab-colors' TypeError: Cannot read property 'dispose' of undefined
  at C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:25:28
  at Set.forEach (native)
  at CompositeDisposable.module.exports.CompositeDisposable.dispose (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:24:26)
  at C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:25:29
  at Set.forEach (native)
  at CompositeDisposable.module.exports.CompositeDisposable.dispose (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:24:26)
  at Object.deactivate (C:\Users\Puranjay\.atom\packages\grammar-tab-colors\lib\grammar-tab-colors.js:108:24)
  at Package.module.exports.Package.deactivate (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\package.js:641:21)
  at PackageManager.module.exports.PackageManager.deactivatePackage (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\package-manager.js:622:12)
  at C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\package-manager.js:337:21
  at C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\config.js:631:20
  at Function.module.exports.Emitter.simpleDispatch (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
  at Emitter.module.exports.Emitter.emit (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
  at Config.module.exports.Config.emitChangeEvent (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\config.js:816:29)
  at Config.module.exports.Config.setRawValue (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\config.js:608:19)
  at Config.module.exports.Config.set (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\config.js:227:14)
  at Config.module.exports.Config.pushAtKeyPath (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\config.js:361:12)
  at Package.module.exports.Package.disable (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\package.js:81:26)
  at PackageManager.module.exports.PackageManager.disablePackage (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\src\package-manager.js:179:16)
  at HTMLButtonElement.<anonymous> (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\settings-view\lib\package-card.js:330:27)
  at HTMLButtonElement.jQuery.event.dispatch (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
  at HTMLButtonElement.elemData.handle (C:\Users\Puranjay\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)
```

**Comment by thtliife** [comment]
Fix coming later today

**Comment by thtliife** [comment]
Fixed in v0.1.5
